tests/test_z_cmdline.py

```diff
@@ -635,7 +635,11 @@ def _alwayscopy_not_supported():
     # see: https://github.com/pypa/virtualenv/issues/565
     if hasattr(platform, 'linux_distribution'):
         _dist = platform.linux_distribution(full_distribution_name=False)
-        if _dist[0] == 'centos' and _dist[1][0] == '7':
+        (name, version, arch) = _dist
+        if any((
```

Comment:
what I meant 
``` python
        if any(( name == 'centos' and version[0] == '7',
                 name == 'SuSE' and arch == 'x86_64' )):
```

Reply:
The current state is closer to [this pep8 recommendation](https://www.python.org/dev/peps/pep-0008/#id29) (the first example). This makes suppressions very clear to read in the logs. (1 suppression: 1 line).
But just confirm your preference and I'll update accordingly.

Reply:
I tend to disagree with that, so I would go to change it to be more readable :thinking:

Reply:
Well, then this is a matter of taste ;-) 
So your code was copy/pasted, but failed `fix-lint`. Had to remove few spaces.